x-pack/solutions/security/plugins/security_solution/public/resolver/view/controls/index.tsx

```diff
@@ -143,22 +139,20 @@ export const GraphControls = React.memo(
             isOpen={activePopover === 'nodeLegend'}
             setActivePopover={setActivePopover}
           />
-          {!isDatePickerAndSourcererDisabled ? (
-            <>
-              <SourcererButton
-                id={id}
-                closePopover={closePopover}
-                isOpen={activePopover === 'sourcererSelection'}
-                setActivePopover={setActivePopover}
-              />
-              <DateSelectionButton
-                id={id}
-                closePopover={closePopover}
-                isOpen={activePopover === 'datePicker'}
-                setActivePopover={setActivePopover}
-              />
-            </>
-          ) : null}
+          <>
```

Comment:
the wrapper can be removed as well

Reply:
yup, not sure how I missed that... done!